<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProject</class>
 <widget class="QDialog" name="NewProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Project Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="leProjectName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Project directory</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="leProjectDirectory"/>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pbBrowse">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Use as default project directory</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pbNext">
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pbCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leProjectName</tabstop>
  <tabstop>pbNext</tabstop>
  <tabstop>pbCancel</tabstop>
  <tabstop>leProjectDirectory</tabstop>
  <tabstop>pbBrowse</tabstop>
  <tabstop>checkBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
